libraries/ms-drivers/inc/max6643_fan_controller.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "gpio.h"
+#include "gpio_it.h"
```

Comment:
At the top here between the #pragma once and the start of the includes you need the header comments:
```c
// Brief description of the driver.
// Requires X, Y, and Z to be initialized.
```
You should also note that we're assuming the high/low temperature threshold selection pins are set in hardware, as is the full speed pin. (I agree with the decision not to include full speed pin handling in the driver since the nand-gate logic is part of the solar board rather than the IC, that logic will go in solar later.)